generator/collector_routing_city_boundaries.cpp

```diff
@@ -103,18 +103,18 @@ RoutingCityBoundariesCollector::LocalityData::Deserialize(ReaderSource<FileReade
 // RoutingCityBoundariesCollector ------------------------------------------------------------------
 
 RoutingCityBoundariesCollector::RoutingCityBoundariesCollector(
-    std::string const & filename, std::shared_ptr<cache::IntermediateData> cache)
+    std::string const & filename, std::shared_ptr<cache::IntermediateDataReader> const & cache)
   : CollectorInterface(filename)
   , m_writer(std::make_unique<RoutingCityBoundariesWriter>(GetTmpFilename()))
-  , m_cache(std::move(cache))
```

Comment:
это была рекомендация clang-tidy "Pass by value and use std::move instead of passing by const reference and copy"
я ее понимаю так: если тебе в c++ надо отдать объект для копирования, скопируй сразу и пусть дальше класс/ф-ия с ним что угодно делает, то есть вызывающий код сразу владеет объектом, а не использует ссылку на него

но да ладно, это так